database/omni_db_schema.psql

```diff
@@ -120,10 +120,12 @@ create table if not exists AddressStats (
 create table if not exists AddressesInWallets (		/* many-to-many */
 	Address varchar(64) not null		/* Address must exist in the AddressBalances table */
 	, WalletID uuid	not null		/* WalletID must exist in the Wallets table */
+    , Protocol Protocol not null default 'Bitcoin'  /* initially 'Bitcoin' or 'Omni' */
```

Comment:
This is a meaningful (non-whitespace change) - these fields are necessary for the initial DB migration.